### ISSUE TITLE
Remove redundant alternative text from images

### ISSUE THREAD

**Issue by ChiuchiuSorin** [xwikisas/application-filemanager#113]
Inside the file-actions toolbar, the alternative text of the images should be replaced with `""`, as it brings no additional information to the already present image text.
